docs/group-user-management.rst

```diff
@@ -0,0 +1,227 @@
+
+3. In the top toolbar, click the Admin button |image0|\ .
+
+4. Managing Groups:
+
+   a. Click on the Groups tab.
+
+   b. You can search for groups if desired
+
+   c. To create a new Group:
+
+      i.   Click on the Add new Group button
+
+      ii.  The Name and Permissions fields are mandatory.
+
+      iii. Click Save
+
+   d. To edit a Group:
+
+      iv.  Click on the Pencil button |image1|
+
+      v.   You can add or remove members or group’s owners. Change permissions etc..
+
+      vi.  Make sure that the data owned by a user is moved or transferred to another user before removing the user from the group
+
+      vii. Click Save
+
+5. Managing Users:
+
+   e. Click on the Users tab
+
+   f. You can search for users if desired
+
+   g. How to identify users:
+
+      viii. Users with administrators privileges with \ |image2|
+
+      ix.   Active users with\ |image3|
+
+      x.    Inactive users with |image4|
+
+      xi.   LDAP users with |image5|
+
+   h. To create a new User:
+
+      xii.  Click on the Add new User button
+
+      xiii. Mandatory fields are highlighted in red
+
+      xiv.  You can select the role of the user
+
+      xv.   Click Save
+
+   i. To edit a User:
+
+      xvi.   Click on the Pencil button |image6|
+
+      xvii.  You can add or remove the user to/from a group. Modify the roles etc..
+
+      xviii. Click Save
+
+   j. Creating an administrator privileges allow to give some limited rights to some trusted users e.g. to allow a facility manager to import data for other users. It is currently preferable to create users with such role via the Web Interface.
+
+6. Note that a user can manage his/her settings.
+
+   k. In the top-right corner of the webclient, click on your name
+
+   l. In the dropdown menu, click on User settings
+
+   m. In the dialog that pops up, the user can change password, default group i.e. the group he/she will log in by default etc.
+
+Administrate using the Command Line Interface
+---------------------------------------------
+
+#.  Managing Groups:
+
+    a. By default when creating a group, its permissions level is set to private. To create a new read-annotate group Lab1, run:
+
+       ``$ omero group add Lab1 --type=read-annotate``
+
+       or
+
+       ``$ omero group add Lab1 --perms='rwra--'``
+
+    b. To list all the groups and save the output for example in a CSV file:
+
+       ``$ omero group list --style csv > groups.csv``
+
+    c. To add an existing user user-1 to the Lab1 group and make him/her a group owner (the option ``--as-owner`` is not needed when adding a member), run:
+
+       ``$ omero group adduser user-1 --name=Lab1 --as-owner``
+
+    d. Let’s add trainer-1 as an owner of the group too:
+
+       ``$ omero group adduser trainer-1 --name=Lab1 --as-owner``
+
+    e. To remove user-1 from the list of owners (user-1 will still be a member of the Lab1 group):
+
+       ``$ omero user leavegroup Lab1 --name=user-1 --as-owner``
```

Comment:
If I did this without the ```--as-owner``` flag, would it remove the user-1 from the group?

Reply:
> f I did this without the --as-owner flag, would it remove the user-1 from the group?

@will-moore Yes, and the corresponding explanation is added in https://github.com/ome/omero-guide-introduction/pull/1/commits/82a3b9427911c0776b3fd9ef050d43c6133c3e62